vertical-pod-autoscaler/e2e/v1/admission_controller.go

```diff
@@ -449,6 +449,47 @@ var _ = AdmissionControllerE2eDescribe("Admission-controller", func() {
 		}
 	})
 
+	ginkgo.It("accepts valid and rejects invalid VPA object", func() {
+		ginkgo.By("Setting up valid VPA object")
+		validVPA := []byte(`{
+			"kind": "VerticalPodAutoscaler",
+			"apiVersion": "autoscaling.k8s.io/v1",
+			"metadata": {"name": "hamster-vpa-valid"},
+			"spec": {
+				"targetRef": {
+					"apiVersion": "apps/v1",
+					"kind": "Deployment",
+					"name":"hamster"
+				},
+		   	"resourcePolicy": {
+		  		"containerPolicies": [{"containerName": "*", "minAllowed":{"cpu":"50m"}}]
+		  	}
+		  }
+		}`)
+		err := InstallRawVPA(f, validVPA)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "Valid VPA object rejected") //
+
+		ginkgo.By("Setting up invalid VPA object")
+		invalidVPA := []byte(`{
+			"kind": "VerticalPodAutoscaler",
+			"apiVersion": "autoscaling.k8s.io/v1",
+			"metadata": {"name": "hamster-vpa-invalid"},
+			"spec": {
+				"targetRef": {
+		    	"apiVersion": "apps/v1",
+		    	"kind": "Deployment",
+		    	"name": "hamster"
+		   	},
+		   	"resourcePolicy": {
+		  		"containerPolicies": [{"containerName": "*", "minAllowed":{"requests":{"cpu":"50m"}}}]
```

Comment:
I think `minAllowed` is the only difference between the valind and the invalid VPAs. Please leave a comment pointing to the difference

Reply:
Done